.rhdh/docs/openshift.adoc

```diff
@@ -26,6 +26,10 @@ image::images/rhdh_from_operator.png[RHDH from Operator]
 
 === Configurations for Operator-backed RHDH
 
+NOTE: At the moment, updates to the Backstage Custom Resource (CR) are automatically handled by the Operator.
```

Comment:
I do not think current implementation of update is correct, as discussed:
https://github.com/janus-idp/operator/pull/119#discussion_r1452064687

In general, an Operator must not deal with spec fields: 
"Types with both spec and status stanzas can (and usually should) have distinct authorization scopes for them. This allows users to be granted full write access to spec and read-only access to status, while relevant controllers are granted read-only access to spec but full write access to status."

And in this particular case, as stated https://kubernetes.io/docs/concepts/services-networking/cluster-ip-allocation/ if ClusterIP is dynamically assigned by contol plane (the case we are testing) we should not manage it during CR update. 

Thats the reason I asked [this PR](https://github.com/janus-idp/operator/pull/119) to be changed and so, we hardly can declare supporting Update cases.